src/In.ProjectEKA.DefaultHip/Link/Database/Migrations/20200122093241_LinkRequestMigrations.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace In.ProjectEKA.DefaultHip.Link.Database.Migrations
+{
+    public partial class LinkRequestMigrations : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4807294)